src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/helpers/AdSkinHelper.java

```diff
@@ -0,0 +1,105 @@
+import com.wikia.webdriver.common.logging.PageObjectLogging;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Dimension;
+import org.openqa.selenium.Point;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import java.awt.image.BufferedImage;
+import java.io.File;
+
+/**
+ * @author drets
+ * @ownership AdEng
+ */
+public class AdSkinHelper {
+
+  public static final String WIKIA_MESSAGE_BUBLE = "#WikiaNotifications div[id*='msg']";
+  public static final int IMAGES_THRESHOLD_PERCENT = 12;
+  private final WebDriver driver;
+  private final AdsComparison adsComparison;
+  private final String pathToLeftPart;
+  private final String pathToRightPart;
+  private final ImageEditor imageEditor;
+  private final ImageComparison imageComparison;
+  private final Shooter shooter;
+  private final int viewPortWidth;
+  private final int articleRightSideX;
+  private final int articleLeftSideX;
+  private final int startSkinY;
+
+
+  public AdSkinHelper(String pathToLeftPart, String pathToRightPart, WebDriver driver) {
+    this.pathToLeftPart = pathToLeftPart;
+    this.pathToRightPart = pathToRightPart;
+    this.driver = driver;
+    this.adsComparison = new AdsComparison();
+    this.imageEditor = new ImageEditor();
+    this.shooter = new Shooter();
+    this.imageComparison = new ImageComparison();
+    WebElement wikiaArticle = driver.findElement(By.id("WikiaPageBackground"));
+    WebElement globalNavigation = driver.findElement(By.id("globalNavigation"));
+    this.viewPortWidth = globalNavigation.getSize().getWidth();
+    this.startSkinY = globalNavigation.getSize().getHeight();
+    this.articleLeftSideX = wikiaArticle.getLocation().x;
+    this.articleRightSideX = wikiaArticle.getLocation().x + wikiaArticle.getSize().getWidth();
+  }
+
+  public boolean skinPresent() {
+    return skinPartPresent(true) & skinPartPresent(false);
+  }
+
+  private boolean skinPartPresent(boolean isLeft) {
+    hideCoveredSkinElements();
+    String pathToFile = isLeft ? pathToLeftPart : pathToRightPart;
+    File patternFile = new File(pathToFile);
+    BufferedImage patternImage = imageEditor.fileToImage(patternFile);
+    if (shouldCutPattern(patternImage.getWidth(), isLeft)) {
+      patternFile = getCutPattern(patternImage, isLeft);
+      patternImage = imageEditor.fileToImage(patternFile);
+    }
+    Dimension size = new Dimension(patternImage.getWidth(), patternImage.getHeight());
+    PageObjectLogging.logImage("EXPECTED", patternFile, true);
+    Point startPoint = getStartPoint(patternImage, isLeft);
+    File actualFile = shooter.capturePageAndCrop(startPoint, size, driver);
+    PageObjectLogging.logImage("ACTUAL", actualFile, true);
+    BufferedImage actualImg = imageEditor.fileToImage(actualFile);
+    return !imageComparison.areImagesDifferent(patternImage, actualImg, IMAGES_THRESHOLD_PERCENT);
+  }
+
+  private Point getStartPoint(BufferedImage patternImage, boolean isLeft) {
+    return isLeft ?
```

Comment:
A ternary operator here should be replaced for more visibility, can you please do it?